releasenotes/notes/clib-grover-op-cb032144e899ed0d.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features_circuits:
+  - |
+    Added :func:`.grover_operator` to construct a Grover operator circuit, used in e.g.
+    Grover's algorithm and amplitude estimation/amplification. This function is similar to
+    :class:`.GroverOperator`, but does not require choosing the implementation of the
+    multi-controlled X gate a-priori and let's the compiler choose the optimal decomposition
+    instead.
```

Comment:
Given the amount of examples you have in the docstring, I would copy-paste one into the reno for quick reference.